creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -347,27 +380,37 @@ class SnapbackSM {
         newReplicaSetSPIds.slice(1) // [secondary1, secondary2]
       )
 
+      response.issuedReconfig = true
+
       // Sync data from existing primary to new secondary
       phase = issueUpdateReplicaSetOpPhases.ENQUEUE_SYNCS
+      await this.enqueueSync({
+        ...baseSyncRequestParams,
+        primaryEndpoint: primary,
+        secondaryEndpoint: newSecondary1
+      })
       await this.enqueueSync({
         ...baseSyncRequestParams,
         primaryEndpoint: primary,
         secondaryEndpoint: newSecondary2
       })
     } catch (e) {
```

Comment:
eh I dont feel strongly. this will give us good insights if we need to examine any particular users. however, I can forsee a lot of logs, especially since issuing reconfigs is recurring + occurs on user triggered events. also, we log out erroneous states and a decisionTree with the steps performed. 

wdyt

Reply:
i don't foresee a lot of logs here as these events should be very infrequent (if they aren't then we def want the logs and need to fix soething) and also i don't think we log out anywhere else the successful Reconfig ops

the current logging structure is:
- log out projected Reconfig op
- if !issueReconfig, log out that this will be skipped
- if Reconfig op fails anywhere, log error

In this current structure it is pretty difficult to parse logs to find which ones errored that should have gone through vs which ones went through. i think it could be simplified to

- if skip, log Reconfig op with skip status
- if success, log Reconfig op with success status
- if error, log Reconfig op with error status

this way the core msg string is consistent across all 3 and is only logged once per op

Reply:
sure, will add